### DISCUSSION TITLE
OpenAI extension: using another embedding

### DISCUSSION THREAD

**Discussion by Simplegram** [oobabooga/text-generation-webui#3055] [Q&A]
Hello, can I use a different embedding other than all-mpnet-base-v2 and all-MiniLM-L6-v2? I am trying to make the API work for Quivr and it needs an embedding with 1536 dimension just like ada-002.

**Comment by matatonic** [comment]
Yes, you can update the model to use another by setting the environment variable OPENEDAI_EMBEDDING_MODEL. 
I don't know off hand of a model that is exactly 1536 in size however and just to be clear, embeddings created with one model cannot be use with another. For Quivr however, I think it would be easiest to patch the database schema to be  the size of the vector you're using: https://github.com/search?q=repo%3AStanGirard%2Fquivr%20VECTOR(1536)&type=code and then recreate the database.
